test_fixtures/task_runner/failing_tasks/lib/main.dart

```diff
@@ -0,0 +1,15 @@
+List longList = [
+  'one',
+  'two',
+  'three', 'four',
```

Comment:
this is to cause the task to fail to verify that a failed task ran fails the parent task